src/SMAIAXBackend.Domain/Model/Entities/SmartMeter.cs

```diff
@@ -1,20 +1,36 @@
 using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography.X509Certificates;
 
+using SMAIAXBackend.Domain.Model.ValueObjects;
 using SMAIAXBackend.Domain.Model.ValueObjects.Ids;
 
 namespace SMAIAXBackend.Domain.Model.Entities;
 
 public sealed class SmartMeter : IEquatable<SmartMeter>
 {
     public SmartMeterId Id { get; } = null!;
+
+    public ConnectorSerialNumber ConnectorSerialNumber { get; private set; } = new ConnectorSerialNumber(Guid.Empty);
```

Comment:
Do we need to assign `new ConnectorSerialNumber(Guid.Empty);` here?

Reply:
I think so? Optimally we wouldn't need it but that would mean refactoring the current create smart meter process which will be done in a later step (create is changed to add as the create is already called when a smart meter connector is ordered)